docker/template.conf

```diff
@@ -0,0 +1,105 @@
+include "metadata.conf"
+
+conseil: {
+  hostname: "0.0.0.0"
+  port: {{APIPORT}}
```

Comment:
is this 80 or 1337?

Reply:
The default value as set above in `entrypoint.sh` is 80. If you set the env var `API_PORT` to some other number it will use that instead.

Reply:
Same goes for all the other env-vars usage